mlkem/randombytes.h

```diff
@@ -9,11 +9,21 @@
 #include <stdint.h>
 
 #include "cbmc.h"
+#include "common.h"
 
-void randombytes(uint8_t *out, size_t outlen)
+
+#if defined(MLK_CUSTOM_RANDOMBYTES)
+static MLK_INLINE void mlk_randombytes(uint8_t *out, size_t outlen)
```

Comment:
This declaration is not needed, is it? We don't use `MLK_CUSTOM_RANDOMBYTES` with CBMC, and it's the user's responsibility to provide a declaration.

Reply:
removed.